### ISSUE TITLE
[SC2126] doesn't check for wc's parameters

### ISSUE THREAD

**Issue by andrei-pavel** [koalaman/shellcheck#797]
#### Bug
- Rule Id: SC2126 
- My shellcheck version: 0.3.7
- I tried on shellcheck.net and verified that this is still a problem on the latest commit

#### Here's a snippet or screenshot that shows the problem:

```sh

#!/bin/sh
echo "something else" | grep "some" | wc -w

```

#### Here's what shellcheck currently says:
SC2126: Consider using grep -c instead of grep|wc.

#### Here's what I wanted or expected to see:
nothing in this case since wc -c cannot be worked around with grep.
It should check for wc's parameters and if it has "-l" but doesn't contain "-c", "-m", or "-w", then it should print:
SC2126: Consider using grep -c instead of grep | wc -l.
else print nothing

EDIT: formatting

**Comment by koalaman** [comment]
Fixed by @Scorpiokat in 85e6c35. Thanks!
